NuEVI/config.h

```diff
@@ -21,4 +21,19 @@
 #define CC_INTERVAL 2
 
 
+// MAybe move these to config.h (as defines?)
```

Comment:
It seems to me this is redundant (as those are defines in config.h)

Reply:
Yes, maybe some clean-up would be a good idea. :grin: